net-misc/dleyna-server/dleyna-server-0.5.0.ebuild

```diff
@@ -1,8 +1,8 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2016 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=5
+EAPI=6
```

Comment:
this is a stable ebuild. Leave it alone, and revbump it rather (to `~amd64`, using `ebump -C`).